dvc/cache/ssh.py

```diff
@@ -30,6 +30,7 @@ def _exists(chunk_and_channel):
 
         with self.tree.ssh(path_infos[0]) as ssh:
             channels = ssh.open_max_sftp_channels()
+            # NOTE: is the config-based max workers limit relevant here?
```

Comment:
It is not. We open as many sftp channels as we can per JOBS ssh connections.

Reply:
Great! Thank you!